scripts/import/dbSNP_v2/load_dbsnp.pl

```diff
@@ -129,19 +131,25 @@ sub init_reports {
     die("$config->{'error_file'} already exists. Please rename or delete\n");
   }
 
+  # For GRCh38 the 1000Genomes minor allele needs flipping
```

Comment:
This suggests they all need reverse complimenting

Reply:
A comment has been added